src/docs/release-notes/R30/FullNotes.mdx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+## React Native Component Library
+
+-   Updated Component Library with several changes by utilizing React Native Paper V5 with Material Design V3.
+    -   Typography and IconWrapper have been removed.
+    -   5 new components have been added, namely [Chip](https://github.com/etn-ccis/blui-react-native-component-library/blob/dev/docs/Chip.md), [Grade](https://github.com/etn-ccis/blui-react-native-component-library/blob/dev/docs/Grade.md), [Icon](https://github.com/etn-ccis/blui-react-native-component-library/blob/dev/docs/Icons.md#icon-as-a-component), [IconSwitch](https://github.com/etn-ccis/blui-react-native-component-library/blob/dev/docs/IconSwitch.md) and [Overline](https://github.com/etn-ccis/blui-react-native-component-library/blob/dev/docs/Overline.md).
```

Comment:
you can link to master branch for all the components here

Reply:
@huayunh already mentioned this so ignore my comment but link out to component docs master...
https://brightlayer-ui-components.github.io/react-native/?path=/info/components-documentation--chip